pandas/tests/frame/test_indexing.py

```diff
@@ -2713,6 +2713,65 @@ def test_type_error_multiindex(self):
         assert_series_equal(result, expected)
 
 
+class TestSparseDataFrameMultitype(tm.TestCase):
```

Comment:
these need to go in ``pandas/spare/tests`` (maybe new file inside there if nothing) appropriate)